README.md

```diff
@@ -14,24 +16,38 @@ A tool to record what books are in the library and who borrows them. The app wil
 ### [RuboCop](https://docs.rubocop.org/en/stable/)
 
 1. Add this line to the `Gemfile`
-    ```
-    gem 'rubocop', '>= 1.0', '< 2.0'
-    ```
-    *not sure how to use Gemfile? Read [this](https://bundler.io/v1.15/guides/bundler_setup.html).*
+   ```
+   gem 'rubocop', '>= 1.0', '< 2.0'
+   ```
+   _not sure how to use Gemfile? Read [this](https://bundler.io/v1.15/guides/bundler_setup.html)._
```

Comment:
- [ ] Please add a proper `Getting started` section with all the information needed to clone and run the app.